vision/cloud-client/detect/beta_snippets.py

```diff
@@ -174,6 +174,99 @@ def detect_handwritten_ocr_uri(uri):
 # [END vision_handwritten_ocr_gcs_beta]
 
 
+# [START vision_fulltext_detection_pdf_beta]
+def detect_document_features(path):
+    """Detects document features in a PDF/TIFF/GIF file.
+
+    Args:
+    path: The path to the local file.
+    """
+    from google.cloud import vision_v1p4beta1 as vision
+    client = vision.ImageAnnotatorClient()
+
+    with io.open(path, 'rb') as pdf_file:
```

Comment:
if you want to use `io` here then it should be imported inside the function definition so the code snippet on the documentation page does not miss that import.

Reply:
Turns out the builtin function works just fine. I removed io